projects/plugins/jetpack/extensions/blocks/blog-stats/blog-stats.php

```diff
@@ -95,7 +95,7 @@ function load_assets( $attributes ) {
 		_n( 'hit', 'hits', $stats, 'jetpack' )
 	);
 
-	$label = $attributes['label'] ? $attributes['label'] : $fallback_label;
+	$label = isset( $attributes['label'] ) ? $attributes['label'] : $fallback_label;
```

Comment:
Since we now require PHP 7, we can use the null coalescing operator in situations like these:

```suggestion
	$label = $attributes['label'] ?? $fallback_label;
```